components/builder-api-proxy/habitat/config/habitat.conf.js

```diff
@@ -2,7 +2,6 @@ habitatConfig({
     company_id: "{{cfg.analytics.company_id}}",
     company_name: "{{cfg.analytics.company_name}}",
     cookie_domain: "{{cfg.cookie_domain}}",
-    demo_app_url: "{{cfg.demo_app_url}}",
```

Comment:
How does the demo app URL removal fit into restricting origin creation? It does strike me as something that needs to be cleaned up and removed though.

Reply:
It was displayed on the "My Origins" page if you had no origins and the mockup removed it. I looked at the url we had and it just redirected to the chef habitat home page so I just stripped it from the code since it was only in a few spots.